test/test.answers

```diff
@@ -1,296 +1,80 @@
 6073f5fde626e3e9ab2d8ad1c0925f174da8874d6dd60d38a242a902fc622ada  test/results/test0000-aarch64-binary
 2521e21a521f55751f6e36d7514d375dc6066f96188d70586995747f83d0ce16  test/results/test0000-amd64-binary
-868f9c7ddb784912bf562a62c0fe531671c6375ccd6f606099d677661b52664c  test/results/test0000-armv7l-binary
-b4dfdb3f7cef6571968fadd3e586a19226a546b3d470cafb8b7c556e1d2d3bfd  test/results/test0000-knight-native-binary
-235cb8354a6f890c029b61d3371db6c539d22a8fedfc555f171a742def20e1a9  test/results/test0000-knight-posix-binary
-02255d2a702b879a5745dbfe6737772b7f8d6c9ba6b906da2158640bac53564a  test/results/test0000-riscv32-binary
-734dd8ba723a9ed1655824e6158406c0164dd180335ee7f725ea07994b3d5cc7  test/results/test0000-riscv64-binary
-be8111d178501a86934f10de64f815e6812c5ee31ca31eb5a0e51cdabae5ad14  test/results/test0000-x86-binary
```

Comment:
Any chance you can update other test answers too?

Reply:
Force pushed to keep it in one commit. It should have all the checksums now. I don't need to run anything other than `make clean test && make Generate-test-answers && sha256sum test/test1000/proof > test/test1000/proof.answer`, right?

Reply:
As far as I remember that should work.